### DISCUSSION TITLE
GitHub - Copilot

### DISCUSSION THREAD

**Discussion by thomas7sea** [community/community#30971] [GitHub Education] [answered]
Hi, is GitHub co-pilot simply available ?

**Comment by thomas-ayissi** [comment, marked as answer]
Hi, 
Yes, it's simple to get GitHub Co-Pilot. This IA is free if you are student.
Please see :  
1/ Here : https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/
2/ Also here : https://dev.to/twizelissa/how-to-enable-github-copilot-for-free-as-student-4kal
